src/protocols/hpr_protocol_router.erl

```diff
@@ -1,12 +1,75 @@
 -module(hpr_protocol_router).
 
+-include("../grpc/autogen/server/packet_router_pb.hrl").
```

Comment:
Is this import needed?

Reply:
Good eye. The code that needed this moved to `hpr_packet_up.erl`. Removed.